vortex-file/src/lib.rs

```diff
@@ -196,9 +195,9 @@ mod forever_constant {
     }
 }
 
-pub use dispatcher::*;
 pub use forever_constant::*;
 pub use read::*;
+pub use vortex_io::dispatcher::*;
```

Comment:
I don't think we should re-export the dispatcher from this crate? Which I guess also means we shouldn't "forward" the features and instead the user should take a direct dependency on vortex-io and enable what they want.